webpack/assets/javascripts/react_app/components/common/forms/FormAutocomplete/FormAutocomplete.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormGroup, Col, ControlLabel } from 'patternfly-react';
+import { translate as __ } from '../../../../common/I18n';
+import AutoComplete from '../../../AutoComplete';
+
+const FormAutocomplete = ({
```

Comment:
Would it make sense to use the [`FormField`](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/forms/FormField.js) and register `AutoComplete` there?